grid/GridModel.js

```diff
@@ -42,8 +42,10 @@ export class GridModel {
      * @param {BaseStore} store - store containing the data for the grid.
      * @param {Object[]} columns - collection of column specifications.
      * @param {StoreSelectionModel} [selection] - selection model to use
+     * @param {boolean} [emptyText] - empty text to display if grid has no records. Can be valid HTML.
```

Comment:
Great change and cleanup, thank you!  Only tweak is to declare this a string in the doc comment.